packages/api/src/routes/medical/patient.ts

```diff
@@ -310,6 +314,46 @@ router.post(
   })
 );
 
+/**
+ * GET /patient/:id/medical-record
+ *
+ * Returns the url to download a patient's medical record summary, if it exists.
+ *
+ * @param req.cxId The customer ID.
+ * @param req.param.id The ID of the patient whose data is to be returned.
+ * @param req.query.conversionType Indicates how the medical record summary should be rendered. Accepts "pdf" or "html".
+ */
+router.get(
+  "/:id/medical-record",
+  asyncHandler(async (req: Request, res: Response) => {
```

Comment:
Since we have the status endpoint to check if it exists shouldnt we only return the medical record url here? And if it doesnt exists throw a not found error?